src/main/res/values-bn-rIN/strings.xml

```diff
@@ -0,0 +1,190 @@
+    <string name="message_connection_error">সংযোগ ত্রুটি</string>
+    <string name="message_create_and_try_again">ডাউনলোড এর জন্য ফাইলগুলি সন্ধান করুন</string>
+    <string name="message_delete_file_body">এই ক্রিয়াটি পূর্বাবস্থায় ফিরিয়ে আনা সম্ভব না।</string>
+    <string name="message_delete_file_error">ফাইল মোছার সময় কিছু ত্রুটি ঘটছে</string>
+    <string name="message_delete_file_title">মোছার বিষয়টি নিশ্চিত করতে চান?</string>
+    <string name="message_delete_progress">সম্ভব না।</string>
+    <string name="message_duplicate_file_upload">বর্তমান ফাইলটিতেই লিখতে চান?</string>
+    <string name="message_duplicate_file_upload_body">এই ফাইলটি%1$s ইতিমধ্যে বিদ্যমান।\\nআপনি কি উপস্থিত ফাইলটি প্রতিস্থাপন করতে নিশ্চিত?</string>
+    <string name="message_empty_apps">অ্যাপস অনুপস্থিত</string>
+    <string name="message_empty_files">ফাইল অনুপস্থিত</string>
+    <string name="message_empty_recent_files">সাম্প্রতিক ফাইল অনুপস্থিত</string>
+    <string name="message_empty_shares">শেয়ার অনুপস্থিত</string>
+    <string name="message_error_account_exists">অ্যাকাউন্ট ইতিপূর্বে আছে।</string>
+    <string name="message_error_amahi_anywhere_app">The Amahi Anywhere অ্যাপটি আপনার সার্ভার এর কোন জায়গার আয়ত্তাধীন নয়।দয়া করে The Amahi Anywhere অ্যাপটি আপনার নেটওয়ার্কে ইনস্টল, চলমান আর আয়ত্তাধীন তা নিশ্চিত করুন।</string>
+    <string name="message_error_authentication">ব্যবহারকারী নাম অথবা পাসওয়ার্ড সঠিক নয়।অনুগ্রহ পূর্বক আবার চেষ্টা করুন।</string>
+    <string name="message_error_authentication_connection">সংযোগ ত্রুটি। আপনার সেটিংস নিরীক্ষণ করে আবার চেষ্টা করুন।</string>
+    <string name="message_error_authentication_field_empty">ব্যবহারকারীর নাম অথবা পাসওয়ার্ড অসম্পূর্ণ রাখা যাবে না</string>
+    <string name="message_error_connection">সংযোগ ত্রুটি</string>
+    <string name="message_error_search_application">আমরা এই ফাইলগুলি জন্য কোন সমর্থিত অ্যাপ্লিকেশন পাইনি।</string>
+    <string name="message_error_video">ভিডিও চালাতে ত্রুটি</string>
+    <string name="message_file_download_complete">ফাইল ডাউনলোড সফল হয়েছে</string>
+    <string name="message_file_upload">আপলোড করতে ফাইল নির্বাচন করুন</string>
+    <string name="message_file_upload_complete">ফাইল আপলোড সফল হয়েছে</string>
+    <string name="message_file_upload_error">ফাইল আপলোড এর সময় কিছু ত্রুটি হচ্ছে</string>
+    <string name="message_file_upload_title">আপনার ফাইল আপলোড করা হচ্ছে</string>
+    <string name="message_logout">সফলভাবে লগ আউট হয়েছে</string>
+    <string name="message_no_files">ডাউনলোডেড ফাইলটি এখানে প্রদর্শিত হয়েছে</string>
+    <string name="message_no_search_result">অনুসন্ধানে কোন ফলাফল পাওয়া যায়নি।</string>
+    <string name="message_notice_create_some">কিছু তৈরি করে পুনরায় চেষ্টা করুন</string>
+    <string name="message_notice_visit_some">কিছু পরিদর্শন করে পুনরায় চেষ্টা করুন</string>
+    <string name="message_offline_file_deleted">অফ্লাইন ফাইল সফলভাবে সরানো হয়েছে</string>
+    <string name="message_progress_file_downloading">ফাইলটি ডাউনলোড হচ্ছে ...</string>
```

Comment:
Please see these triple dots in other values files. Studio must give a warning for placing three dots like this.

Reply:
No, Studio didn't give me any error and even in default language and other string values this three dots are like this.
See..
![image](https://user-images.githubusercontent.com/71203077/94273063-8a92ab80-ff61-11ea-895f-76ddb61fa47f.png)

Reply:
In other files, it's represented as one character, but here it's represented as three characters. Also it will be a warning and not an error. Try running linter on this file.

Reply:
Ok. I am tring I will inform you after sometime